src/app/examples/todos/todos.reducer.spec.ts

```diff
@@ -0,0 +1,67 @@
+import * as fromTodo from './todos.reducer';
+
+describe('TodoReducer', () => {
+  it('should return the default state', () => {
+    const { initialState } = fromTodo;
+    const action = {} as any;
+    const state = fromTodo.todosReducer(undefined, action);
+
+    expect(state).toBe(initialState);
+  });
+
+  it('should add a todo', () => {
+    const TEST_INITIAL_STATE: fromTodo.TodosState = {
+      items: [],
+      filter: 'ALL'
+    };
+    const action = new fromTodo.ActionTodosAdd({ name: 'Mercuccio' });
+    const state = fromTodo.todosReducer(TEST_INITIAL_STATE, action);
+
+    expect(state.items.length).toEqual(1);
+    expect(state.items[0].name).toEqual('Mercuccio');
+  });
+
+  it('should toggle selected todo', () => {
+    const TEST_INITIAL_STATE: fromTodo.TodosState = {
+      items: [{ id: '1', name: 'Tibald', done: false }],
+      filter: 'ALL'
+    };
+    const isDone = TEST_INITIAL_STATE.items[0].done;
```

Comment:
I guess `isDone` is not really necessary and we can just test for hardcoded `true` instead of more complicated `.not.toEqual`